tests/HashTest.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace LanSuite\Tests;
+
+use LanSuite\PasswordHash;
+use PHPUnit\Framework\TestCase;
+use ReflectionClass;
+
+use function PHPUnit\Framework\assertEquals;
+
+class HashTest extends TestCase
+{
+
+    protected function setUp() : void
+    {
+        $GLOBALS['cfg'] = [];
+          error_reporting(E_ALL);
+
+        // the following test hashes will be used for all functions and tests, thus defined once and shared globally
+        $GLOBALS['hashTestArray'] =
```

Comment:
Lets use data providers for this: https://phpunit.de/manual/3.7/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.data-providers